synapse/storage/databases/state/store.py

```diff
@@ -400,7 +400,7 @@ async def store_state_group(
         room_id: str,
         prev_group: Optional[int],
         delta_ids: Optional[StateMap[str]],
-        current_state_ids: StateMap[str],
+        current_state_ids: Optional[StateMap[str]],
```

Comment:
please can you update the docstring to say what the acceptable combinations are here?